app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -425,6 +425,49 @@ ShortFormApplicationService = (
       return
     )
 
+
+  # ----- TODO: call this function via AccountController and ShortFormApplicationController
```

Comment:
@chrisdolendo yes technically, it's being removed in the next feature story where the `TODO`'s get implemented (that's the one I still have in flight)